crates/core/tedge_mapper/src/az/mapper.rs

```diff
@@ -17,6 +18,12 @@ impl AzureMapper {
     pub fn new() -> AzureMapper {
         AzureMapper {}
     }
+
+    pub async fn init(&mut self, mapper_name: &str) -> Result<(), anyhow::Error> {
+        create_directory_with_user_group("tedge-mapper", vec!["/etc/tedge/operations/az"])?;
+        info!("Initialize tedge mapper {} session", mapper_name);
+        Ok(())
+    }
```

Comment:
With this proposal, we will have to run `tedge_mapper --init az` and `tedge_mapper --init c8y`.

I'm expecting something simpler. Why not create all the operation sub-directories with `tedge init`?